src/modules/auth/auth.service.ts

```diff
@@ -1,21 +1,21 @@
 import { Injectable } from '@nestjs/common';
-import { UsersService } from '../users/users.service';
+import { CustomersService } from '../users/customer.service';
 import { JwtService } from '@nestjs/jwt';
 import * as bcrypt from 'bcrypt';
 import { ILoginJWT, IValidateLocalUser } from './interfaces/auth.interface';
 
 @Injectable()
 export class AuthService {
   constructor(
-    private usersService: UsersService,
+    private customersService: CustomersService,
```

Comment:
We mustn't use customer service or admin service directly in any imports. Use userFactory instead

- [ ] rework to userFactory